site/content/api/Tenant-API.md

```diff
@@ -107,7 +107,7 @@ The table below provides an overview of the standard members defined for the JSO
 | *enabled*                | *yes*     | *boolean*     | If set to `false` the tenant is currently disabled. Protocol adapters MUST NOT allow devices of a disabled tenant to connect and MUST NOT accept data published by such devices. |
 | *resource-limits*        | *no*      | *object*      | Any resource limits that should be enforced for the tenant, e.g. the maximum number of concurrent connections and the maximum data volume for a given period. Refer to [Resource Limits Configuration Format]({{< relref "#resource-limits-configuration-format" >}}) for details. |
 | *tenant-id*              | *yes*     | *string*      | The ID of the tenant. |
-| *trusted-ca*             | *no*      | *object*      | The trusted certificate authority to use for validating certificates presented by devices of the tenant for authentication purposes. See [Trusted Certificate Authority Format]({{< relref "#trusted-ca-format" >}}) for a definition of the content model of the object. |
+| *trusted-ca*             | *no*      | *object* or *array*     | The trusted certificate authority(ies) to use for validating certificates presented by devices of the tenant for authentication purposes. The *trusted-ca* property can contain either a single JSON object  or an array of JSON objects. See [Trusted Certificate Authority Format]({{< relref "#trusted-ca-format" >}}) for a definition of the content model of the object. |
```

Comment:
now that we have removed all optional (management) operations from the Tenant API (they have gone into the new HTTP based management API), we should only use an *array* here. It is a different question, if and how the default implementation supports loading existing tenant information from file that still contains a single object (instead of an array) ...
IMHO we should never have an empty array, should we?

Reply:
> IMHO we should never have an empty array, should we?

yes, thats true. so you are saying we should explicitly mention that in the spec?